### PR TITLE
Added all contents

### PR DESCRIPTION
**Pull request by ericleimar**
Copied selected contents from old "imint.se". Some changes in links, layouts etc.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
@simonmika Vem går igenom dessa? Vi bör ha någon slags kontroll att upstream alltid fungerar åtminstone, men behöver nog ingen ordentlig kontroll (stavfel, grammatik, etc.) förrän i slutet.

**Comment by marcusnaslund** [comment]
Pretend that was in English.

File: README.md [hunk 1]
```diff
@@ -17,3 +17,23 @@ Images and other content is placed in static. Create folders corresponding to th
 1. Clone the repository to your computer with ```git clone https://github.com/imintsystems/imint.se --recursive```.
 2. If Hugo is not installed go to https://github.com/spf13/hugo/releases and download the latest release. (v0.14 was used to build this site)
 3. Stand in the root folder of the cloned repository and run ```hugo server -w -t hugo-vidhance-theme``` to start a local webserver hosting the site and that refreshes the site when a file is edited.
+
+## To-do /Eric
```
Comment (marcusnaslund) [inline README.md]: You should keep this to yourself, not make it part of the entire repo.

**Comment by davidhesselbom** [comment]
Note that the TODO will still be there in the repo... forever.

**Comment by marcusnaslund** [comment]
I think we can live with that.

**Comment by davidhesselbom** [comment]
Sure.

**Pull request merged**